### PR TITLE
[ZH] Add Visual Settings

### PR DESCRIPTION
**Pull request by cdwcgt**
completed

### CODE REVIEW

File: wiki/Visual_Settings/zh.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# 视觉设置
+
+![视觉设置菜单](img/VisualSettings-ZH.png "视觉设置菜单")
+
+**视觉设置菜单** 是一个位于游戏界面最下面的隐藏自定义界面。 可以在谱面加载或游戏暂停时使用。 当视觉设置界面显示时, 游戏将不会开始直到你将光标移开视觉设置菜单。
+
+*注意: 视觉设置的的更改会保存在每张谱面上，但是会在关闭osu!后还原设置。 请在[设置](/wiki/options)侧边栏中进行全局（和永久）设置*
```
Comment (cdwcgt) [inline wiki/Visual_Settings/zh.md:7]: 这个关闭osu!后还原设置感觉是错误的，但是[原版](/ppy/osu-wiki/blob/master/wiki/Visual_Settings/en.md)是这样写的，我也就这样翻了（
```suggestion
*注意: 视觉设置的的更改会保存在每张谱面上。 请在[设置](/wiki/options)侧边栏中进行全局（和永久）设置*
```
实际我觉得应该是这样

**Review by cdwcgt — COMMENTED** [review]
It seems that there is something wrong with the explanation here in the original English, but as it is a translation, it will work just as well

File: wiki/Visual_Settings/zh.md (hunk 1 above)
Comment (cdwcgt) [reply wiki/Visual_Settings/zh.md]: https://github.com/ppy/osu-wiki/pull/4585

File: wiki/Visual_Settings/zh.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+*注意: 视觉设置的的更改会保存在每张谱面上，但是部分设置会在关闭osu!后还原。 请在[设置](/wiki/options)侧边栏中进行全局 （和永久） 设置*
+
+或者, 你也可以在游戏中暂停以显示视觉菜单。 但是, 这个行为在[多人游戏](/wiki/Multi)中会被视为退出游戏并退出房间。
+
+## 设置
+
+| 名称 | 效果 | 备注 |
+| --- | --- | --- |
+| `背景暗化` | 使游戏背景变暗 （包括故事板和/或者背景视频） | 休息时间时，会减少30%的暗化 （最高 0%） （可以在设置中禁用）。 *注意: 背景暗化的设置会保存在每张谱面上，但是会在关闭osu!后还原。* |
+| `禁用故事板` | 如果存在故事板，将删除所有故事板元素。 这个选项不不会影响kiai效果和背景视频。 | 如果显示癫痫警告，推荐有癫痫问题的玩家启用这个选项。 如果这个谱面没有故事板，这个选项将被禁用。 |
```
Comment (kj415j45) [inline wiki/Visual_Settings/zh.md:16]:
```suggestion
| `禁用 Storyboard` | 如果存在故事板，将删除所有故事板元素。 这个选项不不会影响kiai效果和背景视频。 | 如果显示癫痫警告，推荐有癫痫问题的玩家启用这个选项。 如果这个谱面没有故事板，这个选项将被禁用。 |
```

**Review by kj415j45 — CHANGES_REQUESTED** [review]

File: wiki/Visual_Settings/zh.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+*注意: 视觉设置的的更改会保存在每张谱面上，但是部分设置会在关闭osu!后还原。 请在[设置](/wiki/Options)侧边栏中进行全局 （和永久） 设置*
+
+或者, 你也可以在游戏中暂停以显示视觉菜单。 但是, 这个行为在[多人游戏](/wiki/Multi)中会被视为退出游戏并退出房间。
+
+## 设置
+
+| 名称 | 效果 | 备注 |
+| --- | --- | --- |
+| `背景暗化` | 使游戏背景变暗 （包括故事板和/或者背景视频） | 休息时间时，会减少30%的暗化 （最高 0%） （可以在设置中禁用）。 *注意: 背景暗化的设置会保存在每张谱面上，但是会在关闭osu!后还原。* |
+| `禁用 Storyboard` | 如果存在故事板，将删除所有故事板元素。 这个选项不不会影响kiai效果和背景视频。 | 如果显示癫痫警告，推荐有癫痫问题的玩家启用这个选项。 如果这个谱面没有故事板，这个选项将被禁用。 |
```
Comment (TPGPL) [inline wiki/Visual_Settings/zh.md:16]: missing Kiai time link, for reference:

```md
| `Disable storyboard` | Remove all storyboard elements. This does not affect [Kiai Time](/wiki/Kiai_time) and the background video, if any. | This is recommended for players with epilepsy issues for when the beatmap displays an epilepsy warning. This option is disabled if there is no storyboard to play. |
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**